src/index.html

```diff
@@ -21,7 +21,441 @@
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-amount">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac"
+            class="card__img"
+            height="134"
+            width="160"
+          />
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__review">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__review-amount">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-amount">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac"
+            class="card__img"
+            height="134"
+            width="160"
+          />
+
+          <h2 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+
+          <p class="card__code">Product code: 195434</p>
+
+          <div class="card__review">
+            <div class="stars">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__review-amount">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+            <p class="card__price-amount">$2,199</p>
+          </div>
+
+          <a
+            href="#"
+            class="card__buy"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="APPLE A1419 iMac"
+            class="card__img"
+            height="134"
+            width="160"
+          />
```

Comment:
The 'img' tag should have both 'alt' and 'title' attributes for better accessibility. The 'title' attribute provides additional information about the image when the user hovers over it.